packages/truffle-artifactor/artifactor.ts

```diff
@@ -1,15 +1,16 @@
-const Schema = require("truffle-contract-schema");
-const fse = require("fs-extra");
-const path = require("path");
-const { writeArtifact, finalizeArtifact } = require("./utils");
+import Schema from "truffle-contract-schema";
+import fse from "fs-extra";
+import path from "path";
+import { writeArtifact, finalizeArtifact } from "./utils";
 const debug = require("debug")("artifactor");
 
-class Artifactor {
-  constructor(destination) {
+export class Artifactor {
+  destination: string;
+  constructor(destination: string) {
     this.destination = destination;
   }
 
-  async save(artifactObject) {
+  async save(artifactObject: object) {
```

Comment:
We can't type this any more specifically than `object`? It should be a truffle-contract Contract

Reply:
I think `artifactObject` is technically not a `ContractObject` until we normalize it. I believe we allow passing any object in there. Maybe that doesn't matter and I should just type it as a `ContractObject`?

Reply:
Nope, you're probably right! Leave it!